app/templates/components/create-session-message.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if shouldShowMessage}}
+  <div class="ui icon info message eight wide column center aligned">
+    <i class="close icon" onclick={{action (mut isMessageVisible) false}} role="button"></i>
+    <i class="info icon"></i>
+    <div class="content">
+      {{t 'Your Speaker Details have been saved, you can now submit a proposal by clicking Submit proposal!'}}
```

Comment:
@uds5501 the issue asks for a different approach. But in my opinion, this is a better way as we are not forcing the speaker to fill in the session details as soon as he/she fills up speaker data.
However @CosmicCoder96 can decide which approach to follow